elide-model-config/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/compile/ElideDynamicEntityCompiler.java

```diff
@@ -125,9 +125,24 @@ public ElideDynamicEntityCompiler(String path, DBPasswordExtractor dbPasswordExt
             classNames.add(PACKAGE_NAME + entry.getKey());
         }
 
-        compiler.useParentClassLoader(
-                new ElideDynamicInMemoryClassLoader(ClassLoader.getSystemClassLoader(),
-                        Sets.newHashSet(classNames)));
+        // Prepare classpath
+        StringBuilder sb = new StringBuilder();
+        Enumeration<URL> resources = Thread.currentThread().getContextClassLoader().getResources("/");
+        while (resources.hasMoreElements()) {
+            URL url = resources.nextElement();
+            String fileName = url.getFile();
+            sb.append(fileName.substring(0, fileName.lastIndexOf("!"))).append(File.pathSeparator);
+        }
+
+        String classpath = sb.toString();
+
+        if (classpath != null && !classpath.equals("")) {
+            log.debug("Classpath = " + classpath);
+            compiler.useOptions("-classpath", sb.toString());
+        }
+
+        compiler.useParentClassLoader(Thread.currentThread().getContextClassLoader());
```

Comment:
There are some strong recommendations to not use the context class loader and instead just this.class.getClassLoader instead:

https://stackoverflow.com/questions/1771679/difference-between-threads-context-class-loader-and-normal-classloader#:~:text=The%20thread%20context%20classloader%20is,currentThread().